assets/parquet/methodology.md

```
@@ -0,0 +1,96 @@
+
+{  
+ "msg": {  
+   "ts": 1,  
+   "status": "error",  
+   "thread_num": 4,  
+   "backtrace": ""  
+ }  
+}
+```
+
+When configuring the Presto coordinator we set the following option in 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)